overview.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+## The Endgame Plan
+
+The Endgame Plan is a proposal to overhaul and improve the ecosystem, governance, and tokenomics of the Maker Protocol. Its primary aim is for the protocol to reach a self-sustainable equilibrium called the Endgame State. In this state, the protocol is intrinsically resilient, and the scope and complexity of the core protocol will no longer change.
+
+The Endgame Plan was [introduced](https://forum.makerdao.com/t/decentralized-voter-committee-on-tuesday-5pm-cest-all-mkr-holders-welcome/15166) by Rune Christensen, the founder of Maker. After a series of initial discussions, the [MIP set](https://vote.makerdao.com/polling/QmTmS5Nf) ratifying this plan was accepted by Maker Governance on 24 October 2022. 
+
+The latest version describing the Endgame Plan is available [here](https://forum.makerdao.com/t/endgame-plan-v3-complete-overview/17427). 
+
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Goals
+
+The Endgame Plan has multiple goals which come together to create the Endgame State. A non-exhaustive list of these goals is presented below.
```

Comment:
There are a lot of things, but agree that he hasn't said so much about the entirety of the deign aims that we can't include it. Just that it will be a bit much if we go into similar things for all the modules.